src/DependencyInjection/LeagueOAuth2ServerExtension.php

```diff
@@ -129,14 +129,14 @@ private function configureAuthorizationServer(ContainerBuilder $container, array
     {
         $authorizationServer = $container
             ->getDefinition(AuthorizationServer::class)
-            ->replaceArgument('$privateKey', new Definition(CryptKey::class, [
+            ->replaceArgument(3, new Definition(CryptKey::class, [
```

Comment:
I replaced `$foo` with ints because classes aren't part of our bundle, so the name could change and the binding won't work.
WDYT?

Reply:
Very good point 👍